flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScan.java

```diff
@@ -230,7 +215,7 @@ public void testSnapshotReads() throws Exception {
     long timestampMillis = table.currentSnapshot().timestampMillis();
 
     // produce another timestamp
-    Thread.sleep(10);
+    waitUntilAfter(10);
```

Comment:
This should be called with `timestampMillis` because you want to wait until it is strictly after the current snapshot's timestamp.

Reply:
Yes, you are right.